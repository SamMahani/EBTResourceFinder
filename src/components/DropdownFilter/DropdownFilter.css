.dropdown {
  margin: 1.5em;
}
