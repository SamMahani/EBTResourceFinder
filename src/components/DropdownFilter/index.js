export { default } from "./DropdownFilter.jsx";
