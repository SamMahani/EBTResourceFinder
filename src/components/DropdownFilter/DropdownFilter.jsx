import React from "react";
import "./DropdownFilter.css";

const DropdownFilter = ({ value, handleChange }) => (
  <select
    className="dropdown"
    onBlur={handleChange}
    value={value}
    onChange={handleChange}
  >
    <option value="markets">market</option>
    <option value="foodbanks">foodbank</option>
    <option value="snapoffices">snapoffice</option>
    <option value="wicoffices">wicoffice</option>
    <option value="stores">store</option>
  </select>
);

export default DropdownFilter;
