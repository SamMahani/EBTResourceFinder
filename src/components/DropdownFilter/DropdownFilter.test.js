import React from "react";
import { render, cleanup } from "react-testing-library";
import "jest-dom/extend-expect";
import DropdownFilter from "./index.js";

afterEach(cleanup);

it("renders", () => {
  const { asFragment } = render(<DropdownFilter />);
  expect(asFragment()).toMatchSnapshot();
});
