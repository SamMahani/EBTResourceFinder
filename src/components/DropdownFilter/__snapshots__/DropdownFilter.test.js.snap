// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <select
    class="dropdown"
  >
    <option
      value="markets"
    >
      market
    </option>
    <option
      value="foodbanks"
    >
      foodbank
    </option>
    <option
      value="snapoffices"
    >
      snapoffice
    </option>
    <option
      value="wicoffices"
    >
      wicoffice
    </option>
    <option
      value="stores"
    >
      store
    </option>
  </select>
</DocumentFragment>
`;
